src/Storage/Storage/Storage.generated.format.ps1xml

```diff
@@ -76,16 +72,12 @@
                 <Alignment>Left</Alignment>
                 <ScriptBlock>$_.ApproximateMessageCount</ScriptBlock>
               </TableColumnItem>
-              <TableColumnItem>
-                <Alignment>Left</Alignment>
-                <ScriptBlock>$_.EncodeMessage</ScriptBlock>
-              </TableColumnItem>
             </TableColumnItems>
           </TableRowEntry>
         </TableRowEntries>
       </TableControl>
       <GroupBy>
-        <ScriptBlock>$_.CloudQueue.ServiceClient.BaseUri</ScriptBlock>
+        <ScriptBlock>$_.QueueClient.Uri</ScriptBlock>
```

Comment:
Same, Here should not be the queue Uri, but the storage account Queue endpoint. The goal is to make the queue from same account be in a group. 
If it's not easy to get the account queue endpoint, you can also try to group the queue with account name. 

Currently, each queue has a group, the PSH console output is not easy to read.

Reply:
updated to account name.